kit/Kit.cpp

```diff
@@ -3126,6 +3163,33 @@ void runKitLoopInAThread()
 
 #endif // !BUILDING_TESTS
 
+void consistencyCheckJail()
+{
+    static size_t warned = 0;
+    if (!(warned++))
+    {
+        bool failedTmp, failedLo, failedUser;
+        FileUtil::Stat tmp("/tmp");
+        if ((failedTmp = (!tmp.good() || !tmp.isDirectory())))
+            LOG_ERR("Fatal system error: Kit jail is missing its /tmp directory");
+
+        FileUtil::Stat lo(InstDirPath + "/unorc");
+        if ((failedLo = (!lo.good() || !lo.isFile())))
+            LOG_ERR("Fatal system error: Kit jail is missing its LibreOfficeKit directory at '" << InstDirPath << "'");
+
+        FileUtil::Stat user(UserDirPath);
+        if ((failedUser = (!user.good() || !user.isDirectory())))
+            LOG_ERR("Fatal system error: Kit jail is missing its user directory at '" << UserDirPath << "'");
+
+        if (failedTmp || failedLo || failedUser)
```

Comment:
Also return, otherwise we'd report an error and then say "Passed system consistency check".

Before returning, this is where we'd want to have `warned = true;`.